.github/workflows/ci.yml

```diff
@@ -100,25 +104,29 @@ jobs:
           toolchain: ${{ env.MINIMUM_SUPPORTED_RUST_VERSION }}
           components: clippy
 
-      # We don't currently have features, so there's no need to run these checks
-      # - name: cargo clippy (no default features)
-      #   run: cargo clippy --workspace --lib --bins --no-default-features -- -D warnings
+      - name: cargo clippy (no default features with libm)
+        run: cargo clippy --workspace --lib --bins --no-default-features --features libm -- -D warnings
 
-      # - name: cargo clippy (no default features) (auxiliary)
-      #   run: cargo clippy --workspace --tests --benches --examples --no-default-features -- -D warnings
+      - name: cargo clippy (no default features with libm) (auxiliary)
+        run: cargo clippy --workspace --tests --benches --examples --no-default-features --features libm -- -D warnings
 
       - name: cargo clippy (default features)
         run: cargo clippy --workspace --lib --bins -- -D warnings -A clippy::doc_markdown
 
       - name: cargo clippy (default features) (auxiliary)
         run: cargo clippy --workspace --tests --benches --examples -- -D warnings -A clippy::doc_markdown
 
-      # As above
-      # - name: cargo clippy (all features)
-      #   run: cargo clippy --workspace --lib --bins --all-features -- -D warnings
+      - name: cargo clippy (all features)
+        run: cargo clippy --workspace --lib --bins --all-features -- -D warnings
+
+      - name: cargo clippy (all features) (auxiliary)
+        run: cargo clippy --workspace --tests --benches --examples --all-features -- -D warnings
+
+      - run: rustup target add armv7a-none-eabi
```

Comment:
I'd probably add this to the step which adds the rust toolchain, similarly to the WASM task

Reply:
Should be fixed in [241ddb4](https://github.com/linebender/peniko/pull/23/commits/241ddb40a1231170d4312227947bf680f3abb98b)